.github/workflows/master.yml

```diff
@@ -26,29 +26,13 @@ jobs:
     needs: bump-version
     steps:
       - uses: actions/checkout@v2
-        with:
-          fetch-depth: '0'
-      - name: Push Docker Image to Github Registry
-        uses: whoan/docker-build-with-cache-action@v5
-        with:
-          username: "${{ github.actor }}"
-          password: "${{ secrets.GITHUB_TOKEN }}"
-          image_name: ${{ secrets.flytegithub_repo }}/flytepropeller
-          image_tag: latest,${{ github.sha }},${{ needs.bump-version.outputs.version }}
-          push_git_tag: true
-          registry: docker.pkg.github.com
-          build_extra_args: "--compress=true"
-      - uses: engineerd/setup-kind@v0.4.0
+        name: "Checkout"
```

Comment:
Let's keep master.yml as is... this is still needed for master merges.

Reply:
You can do [this trick](https://github.community/t/stop-github-actions-running-on-a-fork/17965/2) to disable running it on master merges on forks

Reply:
ok